content/applications/hr/recruitment/refuse_applicant.rst

```diff
@@ -0,0 +1,137 @@
+=================
+Refuse applicants
+=================
+
+At any point in the recruitment process, an applicant can be refused for a job position. To refuse
+an applicant, click on an applicant's card to navigate to a detailed view of the card. At the top of
```

Comment:
Since this is a new doc, I included some detailed navigational instructions on where the user should go (: Feel free to alter the instructions! I just took a guess on where they should go; as the HR expert, you would know better :blush: 
```suggestion
At any point in the recruitment process, an applicant can be refused for a job position. To refuse
an applicant, start by going to the :menuselection:`Recruitment app --> Applications --> All Applications`. On the :guilabel:`Applications` list, click the (undesired) applicant. At the top of
```

Reply:
Great idea- I added the other way to get to there (since there are two).